hoomd/md/BendingRigidityMeshForceCompute.h

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) 2009-2024 The Regents of the University of Michigan.
+// Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+#include "hoomd/ForceCompute.h"
+#include "hoomd/MeshDefinition.h"
+
+#include <memory>
+
+#include <vector>
+
+/*! \file BendingRigidityMeshForceCompute.h
+    \brief Declares a class for computing bending rigidity energy forces
+*/
+
+#ifdef __HIPCC__
+#error This header cannot be compiled by nvcc
+#endif
+
+#include <pybind11/pybind11.h>
+
+#ifndef __BENDINGRIGIDITYMESHFORCECOMPUTE_H__
+#define __BENDINGRIGIDITYMESHFORCECOMPUTE_H__
+
+namespace hoomd
+    {
+namespace md
+    {
+struct rigidity_params
+    {
+    Scalar k;
+
+#ifndef __HIPCC__
+    rigidity_params() : k(0) { }
+
+    rigidity_params(pybind11::dict params) : k(params["k"].cast<Scalar>()) { }
+
+    pybind11::dict asDict()
+        {
+        pybind11::dict v;
+        v["k"] = k;
+        return v;
+        }
+#endif
+    }
+#ifdef SINGLE_PRECISION
```

Comment:
```suggestion
#if HOOMD_LONGREAL_SIZE == 32
```
`SINGLE_PRECISION` no longer exists.